.eslintrc.js

```diff
@@ -27,6 +27,14 @@
         "airbnb",
     ],
     "rules": {
+        "no-new": [0],
+        "class-methods-use-this": [0],
```

Comment:
Probably it is necessary to make methods as static if they don't use this?

Reply:
I thought about it. But there are many methods which is don't use "this" and if we do it static, they will become public available. I think it is not better way.